packages/material-ui-envinfo/package.json

```
@@ -6,7 +6,7 @@
   },
   "license": "MIT",
   "description": "Logs infos about the environment relevant to @material-ui/*",
-  "version": "1.1.6",
+  "version": "1.1.7",
```

I would propose we change this one too, only to have fewer versions to have to think about (more surface for mistakes). I recall we had a discussion about it already in the past, but I can't remember why we didn't change it. I mean, I can't remember the downside of changing it.

```suggestion
  "version": "5.0.0-beta.0",
```

---

don't do that. it's not associated with the other packages. Please double check if it even changed.

---

I changed it (tests only) while fixing tests on Windows.

---

Then let's not bump it.